tardis/montecarlo/base.py

```diff
@@ -357,10 +357,10 @@ def run(
         self._initialize_geometry_arrays(model)
 
         self._initialize_packets(
-            model.t_inner.value,
+            model.t_inner,
             no_of_packets,
             iteration,
-            model.r_inner[0],
```

Comment:
Why are we getting rid of `[0]`?

Reply:
The `[0]` selects the first shell, so we are using the inner boundary of the first shell.